src/modules/45-dashboards/pages/DashboardsPage.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { ReactElement } from 'react'
+import { Container } from '@wings-software/uicore'
+import { Editions } from '@common/constants/SubscriptionTypes'
+import LevelUpBanner from '@common/components/FeatureWarning/LevelUpBanner'
+import type { ModuleLicenseDTO } from 'services/cd-ng'
+import { useLicenseStore } from 'framework/LicenseStore/LicenseStoreContext'
+import { useStrings } from 'framework/strings'
+
+import { DashboardsContextProvider } from './DashboardsContext'
+import DashboardsHeader from './DashboardsCommonHeaderPage'
+import css from './home/HomePage.module.scss'
+
+interface DashboardsPageProps {
+  children: ReactElement<any, any>
+}
+
+const isEnterpriseLicense = (licenseInformation: { [key: string]: ModuleLicenseDTO } | Record<string, undefined>) => {
+  return Object.values(licenseInformation).some(license => license && license.edition === Editions.ENTERPRISE)
```

Comment:
Can you just use optional chaining?
```suggestion
  return Object.values(licenseInformation).some(license => license?.edition === Editions.ENTERPRISE)
```